spec/lib/mastodon/accounts_cli_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'mastodon/accounts_cli'
+
+RSpec.describe Mastodon::AccountsCLI do
+  describe '#create' do
+    context 'when required USERNAME and --email are provided' do
+      let(:arguments) { ['tootctl_username'] }
+
+      context 'with USERNAME and --email only' do
+        let(:options) { { email: 'tootctl@example.com' } }
+
+        it 'creates a new user' do
+          expect { described_class.new.invoke(:create, arguments, options) }
+            .to change(User, :count)
+            .by(1)
+        end
+
+        it 'creates a new account' do
+          expect { described_class.new.invoke(:create, arguments, options) }
+            .to change(Account, :count)
+            .by(1)
+        end
```

Comment:
I'd say those are slightly overkill (but really it's a nitpick and other people may disagree) and better tests around this would be to check an account with the right username has been created.